src/app/app.module.ts

```diff
@@ -5,10 +5,10 @@ import { FormsModule } from '@angular/forms';
 import { NgxDatatableModule } from '@swimlane/ngx-datatable';
 
 import { MomentModule } from 'ngx-moment';
-import { NgxUIModule } from '../../projects/swimlane/ngx-ui/src/public_api';
+import { NgxUIModule } from '@swimlane/ngx-ui';
```

Comment:
Wouldn't this make it a bit harder to update the docs?

Reply:
It's still importing directly from source, I updated the paths. See here: https://github.com/swimlane/ngx-ui/pull/304/files#diff-e5e546dd2eb0351f813d63d1b39dbc48L17-L18